src/backend.ts

```diff
@@ -311,14 +321,52 @@ export function setupSuspendResumeHandler(): () => void {
 export function setupFocusChangeHandler(): () => void {
   let appIsStartingUp: boolean = false;
   let lastPid = 0;
+  let lastAppid = 0;
+  let validKeyEvent: SystemKeyEvent | null = null;
+  SteamClient.System.UI.RegisterForSystemKeyEvents((e) => {
+    // Have not found any race condition issues with this approach since the key event fires long before focus change
+    validKeyEvent = e;
```

Comment:
I feel like to be on the safe side, `validKeyEvent` should be forcibly set to null after a delay where the STEAM button has not been pressed. I think the lodash `debounce` function could do the job.
Something like:
```ts
const debouncedValidKeyEventReset = debounce(() => { validKeyEvent = null; }, 1000);
...
if (e.eKey === 0) {
  validKeyEvent = e;
  debouncedValidKeyEventReset();
}
```

Reply:
Added here:

```ts
let keyEventFunction = (e: SystemKeyEvent) => {
    const cancelDebouncedEvent = debounce(() => {
      validKeyEvent = null;
    }, 1000);
    if (e.eKey === 0) {
      // Have not found any race condition issues with this approach since the key event fires long before focus change
      validKeyEvent = e;
      cancelDebouncedEvent();
    }
  };
```